spec/models/mixins/aggregation_mixin_spec.rb

```diff
@@ -22,13 +22,12 @@
     it "calculates from objects" do
       cluster = cluster_2_1_host(hardware_args)
       partial_cluster = cluster.hosts[1..1] + cluster.hosts[2..2]
-      expect(cluster.aggregate_cpu_speed(partial_cluster)).to eq(cpu_speed)
-    end
 
-    it "calculates from ids" do
```

Comment:
as a note, i don't think these are valid specs anymore.

Reply:
drop this

and if possible drop all our code that takes a parameter into here (think only 3 methods?) because this is a hack to to get these aggregates onto more associations without defining tons of gook.

if possible, can we fix this test and aggregate partial clusters? use a where `:id => cluster.hosts[1..2].map(&:id)` or something?

just so we cover not the id part but the methods that are under test. (if it is duplicate, drop it)

Reply:
```
it "calculates from partial cluster" do
      cluster = cluster_2_1_host(hardware_args)
      partial_cluster = Host.where(:id => cluster.hosts[1..2].map(&:id))
      MiqPreloader.preload(cluster, :aggregate_cpu_speed)
      MiqPreloader.preload(partial_cluster, :aggregate_cpu_speed)
      expect(cluster.aggregate_cpu_speed).to eq(cpu_speed*2)
      expect(partial_cluster.first.aggregate_cpu_speed).to eq(cpu_speed)
      expect(partial_cluster.last.aggregate_cpu_speed).to eq(0)
    end
```
I think is what this spec looks like but I'm going to argue it's a duplicate of the one that's on line 26: https://github.com/ManageIQ/manageiq/pull/20149/files#diff-100e7177f3e457d10d6882d34b471ad4R26 so I'm taking them out cause that's testing the partial. 👍 for the `where` and will fix that on all these that remain.